src/app/routes/enrolment/view-requests/view-requests.component.ts

```diff
@@ -16,6 +16,7 @@ import { UserClaimState } from '../../../state/user-claim/user.reducer';
 import * as userSelectors from '../../../state/user-claim/user.selectors';
 import { EnrolmentForm } from '../../registration/enrolment-form/enrolment-form.component';
 import { KeyValue } from '@angular/common';
+import { Claim, ClaimData } from 'iam-client-lib';
```

Comment:
Do we use it somewhere?

Reply:
Nah, i didn't cleaned it up. Let me fix that :)

Reply:
@Harasz Adjusted by adding interface to remove few `any`from view-requests.component and from enrolment-list.component.